spec/features/category_spec.rb

```diff
@@ -1,10 +1,15 @@
 RSpec.feature "category" do
   fixtures :categories
-  let!(:category) { categories(:category) }
+  fixtures :carriers
   fixtures :users
+  
+  let!(:category) { categories(:category) }
+  let!(:carrier) { carriers(:carrier) } 
```

Comment:
Can this be changed like that?

```suggestion
  let!(:carrier) { carriers(:carrier, category: category) } 
```

If so, you won't need the change to the `before` block :smile:

Reply:
yes I had thought it, but how it isn't factory_bot and in other files they used that way, I just followed the pattern.